test/message/test_runner_output.js

```diff
@@ -371,3 +371,15 @@ test('rejected thenable', () => {
     },
   };
 });
+
+test('uncaughtException', async () => {
+  await new Promise(() => {
```

Comment:
The `Promise` here doesn't do anything here, right?
```suggestion
test('uncaughtException', () => {
```

Reply:
this fix is only for the case where the test is unfinished

Reply:
Alright, makes sense. Probably worth adding a comment.